build_logic.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+# build_logic.sh - Performs steps to compile and deploy evals
+# exports: evals war file
+# usage: Build evals for local development. Jenkins should be used to deploy to dev and prod
+
+VM=$VM_USR@$VM_ADDRESS
+LIFERAY_SDK=$LIFERAY_DIR/liferay-plugins-sdk-$LIFERAY_SDK_VERSION
+EVALS_WAR=evals-$LIFERAY_SDK_VERSION.1.war
+COMPILED_WAR=$LIFERAY_DIR/bundles/deploy/$EVALS_WAR
+
+# copy evals files to liferay sdk
+rm -rf $LIFERAY_SDK/portlets/evals
+cp -rf $EVALS $LIFERAY_SDK/portlets
+
+# run ant to compile evals
+echo "################################# Compile ##################################################"
+cd $LIFERAY_SDK
+$ANT_HOME
```

Comment:
What does this line mean?

Reply:
I had issues with trying to force ant to use the correct version in case the user had multiple versions. This line was directly calling the ant executable but I figured out the issue and now the line is just `ant`